src/main/java/views/AcesUp/RegionSelect.html

```diff
@@ -0,0 +1,25 @@
+<link href="assets/css/AcesUp.css" rel="stylesheet">
+<div id="header">
+    <h1>Aces Up</h1>
+</div>
+<br><h2 style="text-align: center">Select Your Region</h2></br>
```

Comment:
This looks good. Having separate buttons for spanish and english makes it easy to read and follow along